pom.xml

```diff
@@ -114,6 +114,11 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>

```

Comment:
As we discussed, lets not add newrelic tracing annotations as part of this story.